src/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import App from './App';
+
+ReactDom.render(<App />, document.getElementById('root'));
```

Comment:
- [ ] **Since You are using react@18, Please make sure to use ``` createRoot ``` Instead of ``` ReactDOM.render ```**

Reply:
Okay, I will implement it.